src/vivarium_gates_nutrition_optimization/components/hemoglobin.py

```diff
@@ -131,9 +131,10 @@ def setup(self, builder: Builder):
             creates_columns=self.columns_created,
             requires_streams=[self.name],
         )
-
+        #TODO We include tracked here as a bandaid, given new RMS essentially requires
```

Comment:
Maybe worth it put the Jira ticket number for reviewing this issue in the comment?